en/flight_controller/pixhawk6x_pro.md

```diff
@@ -5,15 +5,20 @@ PX4 does not manufacture this (or any) autopilot.
 Contact the [manufacturer](https://holybro.com/) for hardware support or compliance issues.
 :::
 
-
-<img src="../../assets/flight_controller/pixhawk6x_pro/pixhawk6x_pro_internal_architecture.jpeg" width="800px" title="Pixhawk6X Pro Internal Architecture" />
+![Pixhawk6X Pro Internal Architecture](../../assets/flight_controller/pixhawk6x_pro/pixhawk6x_pro_internal_architecture.jpeg)
 
 ## Key Design Points
+
 - High-performance ADIS16470 Industrial IMU with high accelerometer dynamic range (±40 g), perfect for accurate motion sensing in demanding UAV applications
-- All New advanced durable vibration isolation material with resonance frequency in the higher spectrum, ideal for industrial and commercial drone applications
+- All new advanced durable vibration isolation material with resonance frequency in the higher spectrum, ideal for industrial and commercial drone applications
 - High performance STM32H753 Processor
 - Ethernet interface for high-speed mission computer integration
 
+## Baseboards
+
+The Pixhawk 6X Pro can be purchased with a number of baseboards (or no baseboard) to suit different use cases and vehicle types, including [Standard v2A](https://docs.holybro.com/autopilot/pixhawk-baseboards/pixhawk-baseboard-v2-ports), [Standard v2B](https://docs.holybro.com/autopilot/pixhawk-baseboards/pixhawk-baseboard-v2-ports), and [Mini](https://docs.holybro.com/autopilot/pixhawk-baseboards/pixhawk-mini-baseboard-ports), which are shown below.
+It can also be used with any other Pixhawk Autopilot Bus (PAB) specification-compliant baseboard, such as the [Holybro Pixhawk Jetson Baseboard](../companion_computer/holybro_pixhawk_jetson_baseboard.md) and [Holybro Pixhawk RPI CM4 Baseboard](../companion_computer/holybro_pixhawk_rpi_cm4_baseboard.md).
+
```

Comment:
FYI I added this because the baseboards felt like they were just floating around with no explanation.